charts/rclone-copy/README.md

```diff
@@ -0,0 +1,74 @@
+# rclone Helm for Kubernetes
+
+## Usage:
+- Create `rclone.conf` (either via `rclone configure` or from `rclone.conf.template`)
+- Adapt `values.yaml` to contain your values.
+- Install helm with: `helm install . -f values.yaml -n "mySource-mySoruceType-myDestType"` in this directory.
+
+## Creating rclone.conf from scratch
+- Download `rclone` (on Mac e.g. `brew install rclone`)
+- Run the following commands:
+```bash 
+cd my-repository
+rclone config --config "rclone.conf"
+```
+- Select `n` for "New remote"
```

Comment:
From here onwards you can replace it with sth like: "Follow the steps in the interactive configuration interface"